packages/ui/src/elements/PublishButton/index.tsx

```diff
@@ -199,6 +199,7 @@ export const PublishButton: React.FC<{ label?: string }> = ({ label: labelProp }
         disabled={!canPublish}
         onClick={defaultPublish}
         size="medium"
+        subMenuOverrideDisabled={canSchedulePublish}
```

Comment:
This should be renamed to `disableSubmenu`, then you'd check if there's an explicit boolean value when you go to set its disabled state. If `undefined`, fallback to the parent's `disabled` prop.

```suggestion
        disableSubmenu={canSchedulePublish}
```

Reply:
Right so it should have been `disableSubmenu={false}` despite `disabled` being true.